packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -169,6 +179,74 @@ describe('TabsOverflow', () => {
         expect(finalLeft).to.be.lessThanOrEqual(initialLeft);
     });
 
+    xit('should scroll up to the last item and back in LTR', async () => {
+        // setUserAgent is not currently supported by Playwright
+        await setUserAgent(
+            'Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148'
+        );
+
+        const el = await renderTabsOverflow({
+            count: 8,
+            size: ElementSizes.L,
+            includeTabPanel: true,
+            dir: 'ltr',
+        });
+        await elementUpdated(el);
+        await setViewport({ width: 360, height: 640 });
+        await nextFrame();
+
+        const tabsOverflow = el.querySelector(
+            'sp-tabs-overflow'
+        ) as TabsOverflow;
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.false;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.true;
+
+        await scrollToEnd(el, RIGHT_BUTTON_SELECTOR, 'ltr');
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.true;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.false;
+
+        await scrollToEnd(el, LEFT_BUTTON_SELECTOR, 'ltr');
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.false;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.true;
+    });
+
+    xit('should scroll up to the last item and back in RTL', async () => {
+        // setUserAgent is not currently supported by Playwright
+        await setUserAgent(
+            'Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148'
+        );
+
+        const el = await renderTabsOverflow({
+            count: 8,
+            size: ElementSizes.L,
+            includeTabPanel: true,
+            dir: 'rtl',
+        });
+        await elementUpdated(el);
+        await setViewport({ width: 360, height: 640 });
+        await nextFrame();
+
+        const tabsOverflow = el.querySelector(
+            'sp-tabs-overflow'
+        ) as TabsOverflow;
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.true;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.false;
```

Comment:
Used `overflowState` for assertions due to the fact the `waitUntil` that allows scroll behavior in unit tests needs a specific condition to stop. Haven't found a way to test for a 'bad actor' like behaviour to see if the scroll can exceed the expected limit.
I'm happy to hear any thoughts/alternatives to this